packages/webapp/src/components/Form/Unit/index.jsx

```diff
@@ -144,7 +145,9 @@ const Unit = ({
             size={1}
             onKeyDown={onKeyDown}
             onBlur={inputOnBlur}
-            onChange={inputOnChange}
+            onChange={(event) => {
+              inputOnChange(event), additionalOnChange?.();
```

Comment:
This seems like an elegant little change with big functionality! My only thought is it should mirror `onChangeUnitOption()` and be passed to the hook instead.

Reply:
You mean the `useUnit` hook? Let me defer to @SayakaOno on that! I thought the hook was specific to unit-changing logic, but you're right that's how the other callback is passed. This pattern I used here is just replicating `<NumberInput />` by the way, and I meant to tag Sayaka anyway to double-check on it 🙂

Reply:
I though it is called useUnit() because that is what the input component is called